Go/Bitcoin_Price_Tracker/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Bitcoin_Price_Tracker
+
+A script used to fetch the live price of 1 BTC in INR, written in GOlang. [Colly package](https://github.com/gocolly/colly) is used as a base for the web scraping and tag search.
+
+
+## Setup instructions
+
+After forking the repository on your local machine, follow these commands on your terminal:
+```
```

Comment:
Can we have code fence attributes here pls

    ```shell
    commands
    ```

Reply:
I am sorry ,but I can you describe in little more detail what I need to change?
Do I write the three commands, just after the three backticks (in that case, the immediate command disappears) or do i need to write "shell commands" before the actual commands?

Reply:
I think [this](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks) will help you

Reply:
Ohhh, my bad, I over thought the issue..